js/app.js

```diff
@@ -0,0 +1,13 @@
+const showMenu = document.querySelector('.hamburger');
+const hideMenu = document.querySelector('.close-btn');
+const navlinks = document.querySelector('.links');
```

Comment:
Since all these classes name you used are single tags, I think it will be better to change their  **classes** to **id**